packages/components/src/DateTimePickers/Date/Input/Input.component.js

```diff
@@ -8,10 +8,7 @@ export default function Input(props) {
 	const { value, inputManagement } = useContext(DateContext);
 
 	return (
-		<InputSizer
-			placeholder={inputManagement.placeholder}
-			inputText={value.date ? value.textInput : ''}
-		>
+		<InputSizer inputText={inputManagement.placeholder}>
```

Comment:
It doesn't change a lot since you don't pass a `props.textInput` anymore, but shouldn't it be `placeholder={inputManagement.placeholder}` ?

Reply:
If I pass the value to prop.placeholder, it will apply oblique font style, the text width measures more, and input field will be wider. so I pass the value to props.inputText, it apply regular font style, the input field will be narrow.